run.sh

```diff
@@ -72,10 +87,12 @@ solana-genesis \
   --bootstrap-storage-pubkey "$dataDir"/leader-storage-account-keypair.json \
   --ledger "$ledgerDir" \
   --operating-mode development
+tar jcfS "$ledgerDir/genesis.tar.bz2" -C "$ledgerDir" genesis.bin rocksdb
 
 abort() {
   set +e
   kill "$drone" "$validator"
+  wait "$validator"
```

Comment:
Needed for `run-sanity.sh`, but propagating the exit status of validator to the caller will generally be handy. After all `run.sh` is a wrapper for validator.